dsl2.nf

```diff
@@ -218,7 +218,8 @@ workflow {
       sv_wf(
         bamFiles, 
         manta_wf.out.manta4Combine, 
-        samplestatistics
+        samplestatistics,
+        snv_wf.out.maf4MetaDataParser
```

Comment:
This may not work well with sub-workflow toggles from line 62-76. We have to think about it to ensure modularity still works as we expected.

Reply:
And I think I didn't catch for example line 206 and 225 before. Same issue. We have to think about our `--workflows` flag.